pages/knowledge/colours-and-contrast/text/README.md

```diff
@@ -0,0 +1,79 @@
+---
+navigation_title: "Colour contrast for text"
+position: 3
+changed: "2010-04-21"
+---
+
+# Colour contrast for text
+
+**There is a general minimal colour contrast level for all kinds of text. As an exception, large text can have slightly lower contrast. This requirement applies to both “real” text and rasterised text on images.**
+
+## Minimal contrast ratio
+
+The [Web Content Accessibility Guidelines (WCAG) specify a minimal contrast ratio](https://www.w3.org/TR/WCAG21/#contrast-minimum) of `4.5:1` against adjacent colour(s) for text in general. An exception is made for large-scale text: It is considered easier to read and can therefore be set in a lower contrast ratio of at least `3:1`.
+
+Large-scale text is defined as:
+
+- Either **bold** text of at least 14 points (or 18.5 pixels).
+- Or regular text of at least 18 points (or 24 pixels).
+
+(see [WCAG definition of “large scale (text)”](https://www.w3.org/TR/WCAG21/#dfn-large-scale))
+
+To illustrate this: the following image shows the full spectrum of possible contrast ratios from `1:1` on the left to `21:1` on the right.
```

Comment:
As any kind of users (e.g. blind ones) must be able to read and understand the full ADG content, we need a more detailed description of the upcoming image.

Previously, it was:

> For example, the following image has a white background and shows a text with a colour gradient overlay from white to black.

I feel that even this can be improved like following:

> For example, the following image has a white background and shows the text "The quick brown fox jumps over the lazy dog". The text has a colour gradient overlay from white ("T") to black ("g").

Reply:
**To Do:** Rewrite.

Reply:
Rewrote it. It’s now much more verbose but should be understandable even without seeing the picture

> To illustrate this: the following image shows the  sentence "The quick brown fox jumps over the lazy dog" written on white background. It has 43 characters. The first character "T" has the colour white. The colour of following characters gets gradually darker, the last character "g" is fully black.
>
> `![Text with colour gradient overlay](_media/lazy-dog.png)`
>
> Somewhere around the "x" (the 19th character) the contrast ratio reaches `3:1`. Somewhere around the "m" (the 23rd character) we arrive at `4.5:1`. The last character has the maximum colour contrast ratio of `21:1`.